protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -24,22 +24,22 @@ enum Scheme {
 //
 //  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
 //    info: {
-//      title: "Echo API";
-//      version: "1.0";
-//      description: ";
+//      title: "Echo API"
```

Comment:
Please keep the semicolons, it's the format created by `buf format`, which we run to format all our protobuf files. See https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/proto/examplepb/a_bit_of_everything.proto#L19 for an example.